README.md

```diff
@@ -61,27 +61,17 @@ Install PostgreSQL / PostGIS and its dependencies,
 
 ### Create the database and role
 <pre>
-$ sudo su - postgres
-$ createdb 'hot_exports_dev'
-$ create role hot with password '<-password->'
+$ sudo -u postgres createuser -s -P hot
+$ sudo -u postgres createdb -O hot hot_exports_dev
 </pre>
 
 You might need to update the <code>pg_hba.conf</code> file to allow localhost connections via tcp/ip or
 allow trusted connections from localhost.
 
-Run <code>$ psql -h localhost -U hot -W hot_exports_dev</code>
-<pre>
-$ ALTER ROLE hot SUPERUSER;
-$ ALTER ROLE hot WITH LOGIN;
-$ GRANT ALL PRIVILEGES ON DATABASE hot_exports_dev TO hot;
-$ CREATE EXTENSION POSTGIS;
-$ CREATE EXTENSION HSTORE;
-</pre>
-
 Create the exports schema
 
 <pre>
-$ CREATE SCHEMA exports AUTHORIZATION hot;
+$ sudo -u postgres psql -d hot_exports_dev -c "CREATE SCHEMA exports AUTHORIZATION hot"
```

Comment:
`public` and `exports` are both valid schemas (the postgres driver is configured to search both here: https://github.com/hotosm/osm-export-tool2/blob/master/core/settings/dev.py#L20)

@dodobas think we should we drop the schema creation to remove another step?